siriuspy/siriuspy/devices/rf.py

```diff
@@ -1140,3 +1170,248 @@ def cmd_kill_beam(self):
                 timeout=self.TIMEOUT_WAIT):
             return [False, 'Could not set RF Amplitude Increase Rate back.']
         return [True, '']
+
+
+class SILLRFPreAmp(_Device):
```

Comment:
Remember to change this class after merging with the branch from lnls-sirius/dev-packages#1016. It will merge without conflict, but it will be buggy.